core/chain-configs/src/client_config.rs

```diff
@@ -441,8 +449,12 @@ pub struct ClientConfig {
     pub header_sync_stall_ban_timeout: Duration,
     /// Expected increase of header head height per second during header sync
     pub header_sync_expected_height_per_second: u64,
-    /// How long to wait for a response during state sync
+    /// How long to wait for a response from centralized state sync
     pub state_sync_timeout: Duration,
```

Comment:
Maybe rename this to `state_sync_external_timeout` to make it clear what is the usecase of this timeout.